website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -0,0 +1,102 @@
+
+## Download and Build the Binary
+
+1. Clone the [consul-k8s-cli](https://github.com/hashicorp/consul-k8s-cli) project.
+1. Navigate to the `consul-k8s-cli` directory and issue the following command to build the tool:
+
+   ```shell-session
+   go build -o bin/consul-k8s
+   ```
+
+## Usage
+
+Consul K8s CLI uses the following syntax:
+
+```shell-session
+consul-k8s <SUBCOMMAND> <OPTIONS>
+```
+
+## Subcommands
+
+You can use the following subcommands with `consul-k8s`.
+
+### `install`
+
+The `install` command installs Consul on Kubernetes.
+
+```shell-session
+consul-k8s install <OPTIONS>
+```
+
+The following options are available.
+
+| Flag                                                                                                                                          | Description                                                                                                                                                                                                                                                                                                                                                                           | Default                                 | Required |
+| --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------- | -------- |
+| `-auto-approve` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Boolean value that enables you to skip the installation confirmation prompt.                                                                                                                                                                                                                                                                                                          | `false`                                 | Optional |
+| `-dry-run`                                                                                                                                    | Boolean value that validates the installation and returns a summary.                                                                                                                                                                                                                                                                                                                  | `false`                                 | Optional |
```

Comment:
I think the -config-file flag is missing from the table https://github.com/hashicorp/consul-k8s/blob/e935f0703f63d9ad7931978ca0b3f97a2aac7b77/cli/README.md#consul-k8s-install